app/models/answer.rb

```diff
@@ -0,0 +1,30 @@
+class Answer < ActiveRecord::Base
+
+  has_many :comments, :as => :commentable
+  has_many :votes, :as => :votable
+  belongs_to :user
+  belongs_to :question
+
+  def self.newest_answers
+  
```

Comment:
You could do this in sql with
select * from answers a where a.updated_at = (
   select max(updated_at) from answers a2 where a2.question_id = a.question_id) ;

I'll look up the AR syntax for it.